sentry/src/main/java/io/sentry/Sentry.java

```diff
@@ -381,8 +381,8 @@ private static boolean initConfigurations(final @NotNull SentryOptions options)
           "DSN is required. Use empty string or set enabled to false in SentryOptions to disable SDK.");
     }
 
-    @SuppressWarnings("unused")
-    final Dsn parsedDsn = new Dsn(dsn);
+    // This creates the DSN object and performs some checks
+    options.getParsedDsn();
```

Comment:
This is kind-of brittle, as whoever calls `getParsedDsn()` first, may receives an exception. On the other hand it was like this before, so all good for now 😃